composer.json

```diff
@@ -31,18 +31,44 @@
         "phpmd/phpmd": "2.2.*"
     },
     "replace": {
-        "feature_flag": "*"
+        "feature_flag": "self.version"
     },
     "autoload": {
         "classmap": [
             "Classes/"
-        ]
+        ],
+        "psr-4": {
+            "Aoe\\FeatureFlag\\": "Classes/"
+        }
     },
     "autoload-dev": {
         "classmap": [
             "Tests"
         ]
     },
+    "scripts": {
+        "test:unit": [
+            "[ -e 'vendor/bin/phpunit' ] || composer cleanupdate",
+            "vendor/bin/phpunit --colors -v -c vendor/nimut/testing-framework/res/Configuration/UnitTests.xml Tests/Unit"
+        ],
+        "test:functional": [
+            "[ -e 'vendor/bin/phpunit' ] || composer cleanupdate",
+            "mkdir -p typo3conf/ext",
+            "ln -snvf ../../. typo3conf/ext/feature_flag",
+            "vendor/bin/phpunit --colors -v -c vendor/nimut/testing-framework/res/Configuration/FunctionalTests.xml Tests/Functional"
+        ],
+        "test:params": [
```

Comment:
Please remove the test:params block was only used for PoC in the composer meeting.